data_steward/tools/create_rdr_snapshot.py

```diff
@@ -125,8 +125,10 @@ def main(raw_args=None):
     ]
     skip_tables = [
         AOU_DEATH, COPE_SURVEY_MAP, PID_RID_MAPPING,
-        QUESTIONNAIRE_RESPONSE_ADDITIONAL_INFO
-    ]
+        QUESTIONNAIRE_RESPONSE_ADDITIONAL_INFO, 'consent_validation',
+        'wear_consent'
```

Comment:
Suggestion. Add `wear_consent` to common.py (line 125?) and use common.EHR_CONSENT_VALIDATION for `consent_validation`

Reply:
sure